third-party/CMakeLists.txt

```diff
@@ -134,11 +134,13 @@ include(ExternalProject)
 
 if(WIN32)
   # "nvim" branch of https://github.com/neovim/libuv
-  set(LIBUV_URL https://github.com/neovim/libuv/archive/eeae18d085de25f138c23966f98a179f0fb609e7.tar.gz)
-  set(LIBUV_SHA256 c37d0b7cb1defe69ae8dbb4d712c0d7cf838d6539178e8bcf71c72579ab5b666)
+  # https://github.com/neovim/libuv/pull/10#discussion_r324414258
+  set(LIBUV_URL https://github.com/erw7/libuv/archive/libuv-v1.32.0-test.tar.gz)
+  set(LIBUV_SHA256 SKIP)
 else()
-  set(LIBUV_URL https://github.com/libuv/libuv/archive/v1.30.0.tar.gz)
-  set(LIBUV_SHA256 44c8fdadf3b3f393006a4ac4ba144020673a3f9cd72bed1fbb2c366ebcf0d199)
+  # erw7/update-libuv.
+  set(LIBUV_URL https://github.com/erw7/libuv/archive/2af4cf2.tar.gz)
+  set(LIBUV_SHA256 SKIP)
```

Comment:
is `2af4cf2` intended? The SHA for https://github.com/erw7/libuv/tree/libuv-v1.32.0-test is `e7529ee`.

Reply:
ouch! my mistake.